frontend/src/services/AccountAPI.ts

```diff
@@ -12,3 +12,11 @@ export const createBusinessAccount = async (
 ): Promise<AxiosResponse<any>> => {
   return axios.post(`accounts/business`, businessAccountRequestDTO);
 };
+
+export const getAllRegexEmployeeAccount = async (emailPattern: string): Promise<AxiosResponse<any>> => {
+  return axios.get(`/accounts/employee`, { params: { email: emailPattern } });
+};
+
+export const getAllClientAccount = async (emailPattern: string): Promise<AxiosPromise<any>> => {
```

Comment:
Shouldn't the return be Promise of `AxiosResponse` instead of `AxiosPromise`?

Reply:
true